views/js/qtiCommonRenderer/renderers/interactions/ChoiceInteraction.js

```diff
@@ -43,30 +49,57 @@ define([
     var _pseudoLabel = function(interaction, $container){
 
         $container.off('.commonRenderer');
-        $container.on('click.commonRenderer', '.qti-choice', function(e){
 
-            e.preventDefault();
-            e.stopPropagation();//required toherwise any tao scoped ,i/form initialization might prevent it from working
+        var $radios = $container.find('.qti-choice').find('input:radio,input:checkbox').not('[disabled]').not('.disabled');
 
-            var $box = $(this);
-            var $radios = $box.find('input:radio').not('[disabled]').not('.disabled');
-            var $checkboxes = $box.find('input:checkbox').not('[disabled]').not('.disabled');
+        $radios.on('keydown.commonRenderer', function(e){
+            var keyCode = e.keyCode ? e.keyCode : e.charCode;
+            if(keyCode != KEY_CODE_TAB){
+                e.preventDefault();
+            }
 
-            if($radios.length){
-                $radios.not(':checked').prop('checked', true);
-                $radios.trigger('change');
+            if( keyCode == KEY_CODE_SPACE || keyCode == KEY_CODE_ENTER){
+                _triggerCheckboxes($(this).closest('.qti-choice'));
             }
 
-            if($checkboxes.length){
-                $checkboxes.prop('checked', !$checkboxes.prop('checked'));
-                $checkboxes.trigger('change');
+            var nextInput = $(this).closest('.qti-choice').next('.qti-choice').find('input:radio,input:checkbox').not('[disabled]').not('.disabled');
+            var prevInput = $(this).closest('.qti-choice').prev('.qti-choice').find('input:radio,input:checkbox').not('[disabled]').not('.disabled');
```

Comment:
it is preferable to have jquery container variables starting with `$` like `$prevInput` instead of `prevInput`